MoltenVK/MoltenVK/Commands/MVKCmdTransfer.mm

```diff
@@ -507,7 +507,7 @@ static inline MTLSize mvkClampMTLSize(MTLSize size, MTLOrigin origin, MTLSize ma
         id<MTLTexture> srcMTLTex = _srcImage->getMTLTexture(srcPlaneIndex);
         id<MTLTexture> dstMTLTex = _dstImage->getMTLTexture(dstPlaneIndex);
         if (blitCnt && srcMTLTex && dstMTLTex) {
-			if (cmdEncoder->getDevice()->_pMetalFeatures->nativeTextureSwizzle &&
```

Comment:
Use of `cmdEncoder->getMetalFeatures()` is frequent enough that I wonder if it wouldn't be worth pulling it into a local variable.

Reply:
Thanks. Good catch. I thought I had found all the multiple-uses already. Fixed now.